mgmt/rest/client/client.go

```diff
@@ -86,6 +88,14 @@ func parseURL(url string) error {
 	return nil
 }
 
+// Checks if string is URL
+func isURL(url string) bool {
```

Comment:
Consider moving this function body to single source file (for example `/grpc/common/common.go`) instead of redeclaring it multiple times